test/test_functional_tensor.py

```diff
@@ -795,6 +795,32 @@ def test_solarize2(device, dtype, config, channels):
     )
 
 
+@pytest.mark.parametrize("config", [{"threshold": threshold} for threshold in [0.0, 0.25, 0.5, 0.75, 1.0,]])
+def test_solarize_threshold1_bound(config):
```

Comment:
Why not simplify as follows?
```python
@pytest.mark.parametrize("threshold", [0.0, 0.25, 0.5, 0.75, 1.0,])
def test_solarize_threshold1_bound(threshold):
    # rest goes here
```

Reply:
Thanks for the review! Because I'm not accustomed to pytest, I followed some previous code :)
Let me simplify and send PR again.